object_detection/index.html

```diff
@@ -128,6 +133,13 @@
     integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
     crossorigin="anonymous"></script>
   <script src="https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@3.8.0/dist/tf.min.js" 
```

Comment:
Suggest to use latest tf.js, `https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@latest/dist/tf.min.js`